src/components/tools/edit/EditTool.js

```diff
@@ -150,20 +152,8 @@ function EditToolDialog(props) {
         }
     );
 
-    const handleSubmit = (values, { props }) => {
-        const submission = { ...values };
-
-        //these keys needs to added to submission for interactive tools
-        if (submission.interactive) {
-            const interactive_apps = {
-                cas_url: config.vice.defaultCasUrl,
-                name: config.vice.defaultName,
-                image: config.vice.defaultImage,
-                cas_validate: config.vice.defaultCasValidate,
-            };
-            submission.container.interactive_apps = interactive_apps;
-            submission.container.skip_tmp_mount = true;
-        }
```

Comment:
The `interactive` flag was never set in the form for new tools, so these settings were never added to the submission.